packages/npm-bcrypt/wrapper.js

```diff
@@ -0,0 +1,8 @@
+try {
+  // If you really need the native `bcrypt` package, then you should
+  // `meteor npm install --save bcrypt` into the node_modules directory in
+  // the root of your application.
+  exports.NpmModuleBcrypt = require("bcrypt");
+} catch (e) {
+  exports.NpmModuleBcrypt = require("bcryptjs");
```

Comment:
I guess the question is how often we should warn, since we don't want the warning to be too noisy?